src/app/features/dashboard-widgets/entity-count-dashboard-widget/entity-count-dashboard/entity-count-dashboard.component.html

```diff
@@ -2,7 +2,11 @@
   [icon]="entityIcon"
   theme="child"
   [title]="totalEntities"
-  [subtitle]="label"
+  [subtitle]="subtitle ?? label"
+  [explanation]="
+    explanation ??
+    'Displays the count of entities grouped by the specified fields'
+  "
```

Comment:
Adding a general description here is a great idea. Need to make this translatable also

```suggestion
  [explanation]="
    explanation ??
    'Displays the count of entities grouped by the specified fields'
  "
  i18n-explanation
```